content/en/docs/setup/independent/high-availability.md

```diff
@@ -411,39 +403,48 @@ done
 - Follow [these instructions](/docs/setup/independent/setup-ha-etcd-with-kubeadm/)
    to set up the etcd cluster.
 
-### Copy required files to other control plane nodes
-
-The following certificates were created when you created the cluster. Copy them
-to your other control plane nodes:
-
-- `/etc/kubernetes/pki/etcd/ca.crt`
-- `/etc/kubernetes/pki/apiserver-etcd-client.crt`
-- `/etc/kubernetes/pki/apiserver-etcd-client.key`
+#### Copy required files from an etcd node to all control plane nodes
 
 In the following example, replace `USER` and `CONTROL_PLANE_HOSTS` values with values
 for your environment.
 
 ```sh
+# Make a list of required etcd certificate files
+cat << EOF > etcd-pki-files.txt
```

Comment:
fwiw I'll submit a PR later, but we could bash-fu several of these commands together.

Reply:
This code is intentionally made simple to understand. I value readability to more over fewer complex commands in public documentation.

Reply:
but if your bash-fu is more readable then by all means, submit away